### PR TITLE
Combine PandasLoadOptions into single class

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
We currently have PandasCsvLoadOptions, PandasJsonLoadOptions, PandasNdjsonLoadOptions, and PandasParquetLoadOptions. Which can be confusing to users.

closes: #1681 

## What is the new behavior?
Instead of multiple classes, we can probably merge this in one single class PandasLoadOptions that can be superset for all the PandasLoadOptions.

## Does this introduce a breaking change?
Yes

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/src/astro/dataframes/load_options.py [hunk 1]
```diff
@@ -8,56 +8,30 @@
...
-
-
-@define
-class PandasCsvLoadOptions(PandasLoadOptions):
     """
     Pandas load options while reading and loading csv file.
 
-    :param delimiter: Delimiter to use. Defaults to None
-    :param dtype: Data type for data or columns. E.g. ``{"a": np.float64, "b": np.int32, "c": "Int64"}`` Use str or
+    :param delimiter: valid file type[CSV] - Delimiter to use. Defaults to None
```
Comment (sunank200) [inline python-sdk/src/astro/dataframes/load_options.py:14]: I think we need to have a way to pass all other supported parameters for Pandas as `kwargs` here. We should add this change as part of this PR.

**Review by sunank200 — CHANGES_REQUESTED** [review]
I think the following changes would be required:
1. I think we need to have a way to pass all other supported parameters for Pandas as `kwargs` here. We should add this change as part of this PR. More at: https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html, https://pandas.pydata.org/docs/reference/api/pandas.read_json.html
2. Documentation has to be changed accordingly in `load_file.rst` for this.

**Review by pankajastro — COMMENTED** [review]
In the earlier implementation, I thought a class per pandas API (`PandasJsonLoadOptions`  
 [read_json](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html)) will be cleaner.
currently, you have put a comment like json/csv etc this comment does not look correct to me since `dtype` is a valid param for even read_json method and in future, we may like to add this param in `PandasJsonLoadOptions` class.
I feel we have different classes then the code editor also helps users to understand what parameters they can use, which would be confusing if we have a single class. 
So I would like to know the other thought on this PR

File: python-sdk/src/astro/dataframes/load_options.py (hunk 1 above)
Comment (utkarsharma2) [reply python-sdk/src/astro/dataframes/load_options.py:14]: resolved.

**Comment by utkarsharma2** [comment]
@pankajastro Thanks for pointing this out, I have updated the docstring. Now we are referring to the pandas doc for a superset of all the valid options.

Also, I think if we are anyway exposing to end users that we are using pandas in the backend to do stuff, I think we can also assume that they will be aware of which options need to be passed for a particular type of file. We have also included the pandas link to aid for this.

**Review by tatiana — APPROVED** [review]

**Review by sunank200 — APPROVED** [review]

**Pull request merged**